.gitignore

```diff
@@ -8,6 +8,7 @@
 # Build artifacts
 __pycache__/
 *.py[cod]
+**.eggs
 /*.egg*
```

Comment:
These 2 lines can be just:
```
*.egg*
```

Reply:
Not `**.egg*`? It needs to work in subdirectories too.

Reply:
`*.egg*` is a file match, it will apply to any file (any sub-dir)

`**` is needed in super rare cases, where you want like a very fine grained match.

The doc mentions too: `**/foo` is equivalent to `foo` actually.
The `**` would be needed only if you wanted to do something like `**/foo/bar`

Reply:
The doc BTW: https://git-scm.com/docs/gitignore

Reply:
Ah... I didn't realise "**/foo" was the same as "foo", and the ** syntax is used when you want to match nested structures. Thanks Zoran!